ui/models_test.py

```diff
@@ -45,6 +45,12 @@ def videofile():
     return VideoFileFactory()
 
 
+@pytest.fixture
+def moiralist():
+    """Fixture to create a moira list"""
+    return MoiraListFactory()
```

Comment:
OK, I'll leave to your discretion. Generally I use fixtures for test data when there's a 'tree' of specific hierarchical data that I need, and/or there's a model object I need with some really specific properties. That's just a broad guiding principle, though. If you think we're getting use out of this fixture-izing, I'm for it 👍